### ISSUE TITLE
Use Mouse relative position for SDL trackpad mode

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vgmurphy/limbo-android#21]
```
What steps will reproduce the problem?
1. Boot DSL 4.11 iso
2. Double tap to activate touchpad mode.
3. Move mouse to right most corner of the desktop

What is the expected output? What do you see instead?
Trackpad mouse motions should be interpreted as relative mouse positions so 
they don't get blocked by the boundary checks. Use flag for relative mouse 
position in SDL.




```

Original issue reported on code.google.com by `ideal2i...@gmail.com` on 14 Jan 2013 at 12:20

**Comment by GoogleCodeExporter** [comment]
```
Fix will be tentatively included in version 0.9.5
```

Original comment by `ideal2i...@gmail.com` on 14 Jan 2013 at 12:24

**Comment by GoogleCodeExporter** [comment]
```
Fixed in ver 0.9.5
```

Original comment by `ideal2i...@gmail.com` on 20 Jan 2013 at 11:57
- Changed state: **Done**
